website/build.gradle

```diff
@@ -139,9 +139,14 @@ createBuildTask( name:'Gcs', useTestConfig: true, baseUrl: getBaseUrl(), dockerW
 createBuildTask( name:'Apache', dockerWorkDir: dockerWorkDir)
 
 def getBaseUrl() {
-  System.getenv('ghprbPullId')?.trim() ?:
-    "${System.getProperty('user.name')}-${grgit.branch.current().getName()}"
+  def pullRequestId = System.getenv('ghprbPullId')?.trim()
+  if (pullRequestId) { return pullRequestId }
+
+  // grgit is null if building outside of git (i.e. from source archive)
+  def buildContext = grgit ? grgit.branch.current().getName() : 'archive'
```

Comment:
The reason to base it on the branch is for a pretty URL when you browse? Or is it to avoid clobbering directories on disk, so you can have multiple renderings sitting in `build/` or whatnot? TBH I don't know what grgit is doing in this context (in the others seems obvious, localized, totally sensible).

Reply:
I've added a comment to the method:

```groovy
/**
 * Use the pull request ID in the URL path, or the git branch when building locally.
 * This allows staging multiple work trees without clobbering eachother, i.e.
 * for shared GCS staging, or for locally comparing the differences between
 * branches.
 */
```

Using the branch name locally seems less important, but I see some value in having it. I'm not opposed to removing it for simplicity.